src/Sarif/Writers/BaseLogger.cs

```diff
@@ -7,25 +7,42 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.Writers
 {
-    public class BaseLogger
+    public abstract class BaseLogger
     {
-        private readonly List<FailureLevel> failureLevels;
-        private readonly List<ResultKind> resultKinds;
+        protected readonly List<FailureLevel> _failureLevels;
+        protected readonly List<ResultKind> _resultKinds;
 
         public BaseLogger(IEnumerable<FailureLevel> failureLevels,
```

Comment:
>public [](start = 8, length = 6)

I think we could change the constructor to protected as well, since no one will manually use the constructor. What do u think?

Reply:
Sounds good to me.  Done.

---
In reply to: [560612196](https://github.com/microsoft/sarif-sdk/pull/2249#discussion_r560612196) [](ancestors = 560612196)